src/app/[locale]/(app)/(nav)/overview/_components/UsageStats.tsx

```diff
@@ -21,24 +22,24 @@ export default function UsageStats() {
   const data = [
     {
       value: formatNumber(messageCount(), { notation: "standard" }),
-      label: "messages sent",
+      label: "overview.messagesSent",
```

Comment:
For all of the labels, call `t` after importing it and adding the "use client" directive